client/components/editor.vue

```diff
@@ -358,52 +358,28 @@
           this.unconfirmedCount = this.$editorView.state.collab$.unconfirmed.length;
 
           const sendable = collab.sendableSteps(newState);
-          let addingComment = false;
-          let deletingComment = false;
-
           if (sendable) {
-            if (this.commentHighlightKey) {
-              const commentMarks = _.filter(transaction.steps, (s) => {
-                return s.mark && s.mark.type.name === 'highlight' && s.mark.attrs['highlight-keys'] === this.commentHighlightKey;
+            const containsHighlightStep = sendable.steps.find((x) => {
+              return (x.jsonID === 'addMark' || x.jsonID === 'removeMark') && x.mark && x.mark.type.name === 'highlight';
```

Comment:
Is looking at `jsonID` really the best?

Reply:
Yeah... In serialized steps we could use step.stepType. But now (after deserialization) it seems that we only can use stepID.